addons/settings/script_component.hpp

```diff
@@ -138,6 +139,14 @@
     (uiNamespace getVariable QGVAR(serverTemp))  getVariable [setting, [[setting,  "server"] call FUNC(get), nil]] select 0\
 ] select (["client", "mission", "server"] find TEMP_PRIORITY(setting)))
 
+#define TEMP_VALUE_SOURCE(setting,source) ([\
+    (uiNamespace getVariable QGVAR(clientTemp))  getVariable [setting, [[setting,  "client"] call FUNC(get), nil]] select 0,\
+    (uiNamespace getVariable QGVAR(missionTemp)) getVariable [setting, [[setting, "mission"] call FUNC(get), nil]] select 0,\
+    (uiNamespace getVariable QGVAR(serverTemp))  getVariable [setting, [[setting,  "server"] call FUNC(get), nil]] select 0\
+] select (["client", "mission", "server"] find source))
+
+#define TEMP_VALUE(setting) TEMP_VALUE_SOURCE(setting,TEMP_PRIORITY(setting))
```

Comment:
Now we have TEMP_VALUE() twice.

Reply:
Huh? Somehow I've not staged removal of the old one.

Reply:
Fixed